lib/ood_core/job/adapters/slurm.rb

```diff
@@ -178,6 +179,21 @@ def get_jobs(id: "", owner: nil, attrs: nil)
             return [{ id: id, state: 'undetermined' }]
           end
 
+          def accounts
+            user = Etc.getlogin
+            args = ['-nP', 'show', 'users', 'withassoc', 'format=account,user', 'where', "user=#{user}"]
+            upcase = ENV['OOD_UPCASE_ACCOUNTS'].nil? ? false : true
```

Comment:
I doubt this will happen but if someone says `OOD_UPCASE_ACCOUNTS=false` this would evaluate to `true`. Not sure if that's desirable.